src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -92,6 +94,37 @@ public long getFee() {
     return mOperations;
   }
 
+  /**
+   * Returns returns the claimable balance ID for the CreateClaimableBalanceOperation at the given index within the transaction.
```

Comment:
nit: "Returns returns"

Reply:
Can we also add a cross-reference to the docstring here? Either to the relevant [Stellar Core code](https://github.com/stellar/stellar-core/blob/9f3cc04e6ec02c38974c42545a86cdc79809252b/src/test/TestAccount.cpp#L285) or to the [txnbuild implementation](https://github.com/stellar/go/blob/master/txnbuild/transaction.go#L356). It'll be super helpful if we need to return to this code.